tests/integration/test_vulnerability_detector/test_scan_results/test_alert_vulnerability_removal.py

```diff
@@ -1,181 +1,294 @@
-test_packet_0_name = 'wazuhintegrationpackage-0'
-test_packet_1_name = 'wazuhintegrationpackage-1'
-test_packet_0_cve = 'CVE-000'
-test_packet_1_cve = 'CVE-001'
-
-# Offline feeds
-rhel_oval_feed_path = os.path.join(test_feed_path, vd.CUSTOM_REDHAT_OVAL_FEED)
-rhel_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_REDHAT_JSON_FEED)
-nvd_json_feed_path = os.path.join(test_feed_path, vd.CUSTOM_NVD_FEED)
-parameters = [{
-    'RHEL_FEED_PATH': rhel_oval_feed_path,
-    'RHEL_JSON_FEED_PATH': rhel_json_feed_path,
-    'NVD_JSON_FEED_PATH': nvd_json_feed_path
-}]
-metadata = parameters
-
-# Configuration data
-configurations = load_wazuh_configurations(configurations_path, __name__, params=parameters, metadata=metadata)
-
-# Read JSON data template
-nvd_vulnerabilities = file.read_json_file(nvd_json_feed_path)
-rhel_vulnerabilities = file.read_json_file(rhel_json_feed_path)
+from wazuh_testing.modules import vulnerability_detector as vd
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
 
-# fixtures
-@pytest.fixture(scope='module', params=configurations)
-def get_configuration(request):
-    """Get configurations from the module."""
-    return request.param
 
+# Reference paths
+TEST_DATA_PATH = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+CONFIGURATIONS_PATH = os.path.join(TEST_DATA_PATH, 'configuration_template')
+TEST_CASES_PATH = os.path.join(TEST_DATA_PATH, 'test_cases')
+CUSTOM_FEED_PATH = os.path.join(os.path.dirname(os.getcwd()), 'integration', 'test_vulnerability_detector', 'data')
+
+# Configuration and cases data
+configurations_path = os.path.join(CONFIGURATIONS_PATH, 'scan_result_alert_vulnerability_removal.yaml')
+test_alert_vuln_removal_path = os.path.join(TEST_CASES_PATH, 'cases_alert_vulnerability_removal.yaml')
+
+# Custom feeds path
+rhel_oval_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_REDHAT_OVAL_FEED)
+rhel_json_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_REDHAT_JSON_FEED)
+nvd_json_feed_path = os.path.join(CUSTOM_FEED_PATH, 'feeds', vd.CUSTOM_NVD_FEED)
+
+# Test configurations
+configuration_parameters, configuration_metadata, test_case_ids = configuration.get_test_cases_data(
+    test_alert_vuln_removal_path)
+configurations = configuration.load_configuration_template(configurations_path, configuration_parameters,
+                                                           configuration_metadata)
 
+# Set offline custom feeds configuration
+to_modify = ['RHEL_FEED_PATH', 'RHEL_JSON_FEED_PATH', 'NVD_JSON_FEED_PATH']
+new_values = [rhel_oval_feed_path, rhel_json_feed_path, nvd_json_feed_path]
+configurations = configuration.update_configuration_template(configurations, to_modify, new_values)
+configuration_metadata = configuration.update_configuration_template(configuration_metadata, to_modify, new_values)
+
+
+# fixtures
 @pytest.fixture(scope='function')
-def mock_system(mock_agent_module):
+def setup_log_monitor():
```

Comment:
I think that this fixture is not necessary. You can use a global `FileMonitor` object due to the "logs" are truncated after every test execution